src/main/interface/settings.c

```diff
@@ -355,7 +355,7 @@ static const char * const lookupTableRescueSanityType[] = {
 
 #ifdef USE_MAX7456
 static const char * const lookupTableVideoSystem[] = {
-    "AUTO", "PAL", "NTSC"
+    "AUTO", "PAL", "NTSC", "HDZERO"
```

Comment:
@saidinesh5 , @madchiller , @mricho ,
should we consider `HD720` here rather thn `HDZERO`? (in preparation of HD1080)

Reply:
This video system information is purely specific to HDZero (so as to signify that this OSD supports 50x18 characters) and is used only by the configurator to show the canvas of appropriate size.